lib/typeform_data/client.rb

```diff
@@ -39,10 +39,10 @@ def dump(object)
       Marshal.dump(object)
     end
 
-    # @param serialized [String] The output of Marshal.dump(vci) where vci is either (1) an
+    # @param [String] The output of Marshal.dump(vci) where vci is either (1) an
     #   instance of TypeformData::ValueClass or (2) an array of instances of
     #   TypeformData::ValueClass.
-    # @param default [Object] What to return if 'serialized' is blank or not a String.
```

Comment:
@erose Please don't remove these variable names.

The docs story is different for this project (as opposed to our private app). The docs are actually generated and available here: http://www.rubydoc.info/gems/typeform_data/TypeformData/Typeform so these names are being used.